core/templates/components/cta/cta.twig

```diff
@@ -1,5 +1,31 @@
+{#
+/**
+ * @file
+ * CTA Component.
+ *
+ * Call to action component with clickable image and button.
+ * Variants:
+ *   - Default - Button at the bottom: With full-width button under the image
+ *   - Button in the center: With button in the center of the clickable image
+ *   - Button at the bottom with icon in the center: With full-width button under the image and an icon in the center of the image
+ *
+ * Available variables:
+ * - attributes: For additional HTML attributes not already provided.
+ * - modifier_class: Additional css classes to change look and behaviour.
+ * - src: The href property of the image.
+ * - alt: Alt text for the image.
+ * - link: The href property of the CTA.
+ * - text: The text label of the button.
+ * - icon_src: The href property of the icon image.
+ */
+#}
 <a{{ attributes }} href="{{ link|default("https://stanford.edu") }}" class="su-cta {{ modifier_class }}">
-  <img class="su-cta__img" src="{{ src|default("../../../img/hero-banner.png") }}" alt="{{ alt|default("Please provide alt text") }}">
-  <i class="su-cta__icon {{ icon|default("fas fa-cat") }}"></i>
+  <img class="su-cta__img" src="{{ src|default("../../../img/hero-banner.png") }}" alt="{{ alt|default("Please provide alt text") }}" />
+
+  {# CTA icon #}
+  {% if icon_src is not empty %}
+    <img src="{{ icon_src }}" class="su-cta__icon" />
```

Comment:
This would benefit from an alt tag and `role="presentation"` for accessibility.

Reply:
Very true. Added.